psi4/src/psi4/libfock/jk.cc

```diff
@@ -179,6 +179,12 @@ std::shared_ptr<JK> JK::build_JK(std::shared_ptr<BasisSet> primary, std::shared_
             return build_JK(primary, auxiliary, options, "DISK_DF");
         }
 
+    } else if (jk_type == "DIRECT_DF_LINK") {
+        CompositeJK* jk = new CompositeJK(primary, auxiliary, "DIRECT_DF", "LINK", options);
```

Comment:
Something I want to bring up to the reviewers, that Andy and I discussed yesterday:

Right now, we have the singular CompositeJK type "DIRECT_DF_LINK", with hardcoded J type and K type inputs into CompositeJK. Eventually, given the number of combinations of J and K there will be, it will make the code more extensible to parse the jk_type string using Python tools, extract the J and K types from the jk_type variable, and pass the J and K types in as arguments to the construction of J and K. So, this comes with two questions:
1. Would it be better to implement such a parsing mechanism now, or later on, when more types have been added to CompositeJK?
2. When we implement the parsing mechanism for jk_type, it will be necessary to use characters that clearly separate the J and K types in the jk_type variable; for example, a plus sign could be used, so jk_type would look like "DIRECT_DF+LINK", How do the reviewers feel about using such characters in the SCF_TYPE option?

Reply:
I agree with dissecting the jk_type in Python. No problems anticipated with `+` as delimiter (it's no worse than `/` being the cue for "cbs" or "method/basis" runs). If you don't run into needing the parsing now, I think I'd wait for another case before implementing it -- that'll keep this PR shorter and provide a better test that you're switching as intended.